cmd/consul-registration-hook/main.go

```diff
@@ -96,8 +95,16 @@ var commands = []cli.Command{
 					}
 					err = provider.CheckProbe(context.Background())
 					if err != nil {
-						return fmt.Errorf("Error checking services liveness: %s", err)
+						return fmt.Errorf("error checking services liveness: %s", err)
 					}
+					podTerminating, err := provider.IsPodTerminating(context.Background())
+					if err != nil {
+						return fmt.Errorf("error checking service Termination state: %s", err)
+					}
+					if podTerminating {
+						return fmt.Errorf("wont register, pod in terminating state")
```

Comment:
jako error consul-hooka bym tego nie traktował bo to nie jego wina że POD jest w takim a nie innym statusie więc raczej jako zawsze printowane INFO

Reply:
ale ma przestać działać i zrobić exit, więc zachowanie takie jak error

Reply:
ale zaprzestanie działania (bo jednak nie trzeba wykonać roboty którą się dostało) i exit 0 to nie jest error, tak to widzę 🤷

Reply:
czysto filozoficzne gadanie, dajesz approva  bez tego czy nie?

Reply:
fixed